gemrb/plugins/SDLVideo/Matrix.cpp

```diff
@@ -1,3 +1,4 @@
+#define _USE_MATH_DEFINES
```

Comment:
Isn't this relevant only for MSVC? I think it's better to include win32def.h or better yet, split out the M_PI stuff in a separate header and include that.

Reply:
A valid point, although it's been around for so long I doubt that any other library uses it differently. Those constants should be replaced in C++20, and ideally everyone should use a retroport of the new header.